.github/workflows/main.yml

```diff
@@ -8,15 +9,19 @@ on:
 
 jobs:
   test:
-
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
-        ruby-version: [2.7, 3.0, truffleruby-head]
+        ruby-version:
+          - "2.7"
```

Comment:
I think that we should only keep versions that are still supported (AKA 3.1+ as per https://endoflife.date/ruby)

Maybe also get rid of truffleruby? Can't remember if it was something someone requested compatibility for or if I added just to try and see if it was compatible

Reply:
Yeah, I'm fine to drop older Rubies an EoL'd dependencies in general. Maybe we can do all of that as part of the next major version bump, and include some of the other PRs in it too.